src/Tizen.Multimedia.MediaPlayer/Player/MediaStreamSource.cs

```diff
@@ -33,6 +33,8 @@ public sealed class MediaStreamSource : MediaSource
     {
         private readonly MediaFormat _audioMediaFormat;
         private readonly MediaFormat _videoMediaFormat;
+        private static PlayerHandle _playerHandle;
+        private static IntPtr _handle;
```

Comment:
I think it doesn't need to make `_playerHandle` and  `_handle` to `field`. `Local variable` in property getter is enough.

Reply:
that's right. i'll fix it.